debian/control

```diff
@@ -6,10 +6,11 @@ Standards-Version: 3.9.2
 Build-Depends: debhelper (>= 9), libraspberrypi-dev, libpng12-dev
 
 Package: kano-toolset
-Architecture: all
+Architecture: any
```

Comment:
`all` means that the built package can be installed on any architecture without any changes (e.g. packages only containing python scripts, shell scripts, things which do not need compilation). `any` means that the package can be installed on any architecture but needs to be rebuilt for the platform for this to be possible.